index.html

```diff
@@ -0,0 +1,180 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,400;1,700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css">
+    <title>Capstone Project-Sozo Graphics Institute</title>
+</head>
+<body id="body">
+    <section class="header">   
+        <div class="nav-container">
+            <div class="menu_btn" onclick="openMobileMenu()">
+                <img src="images/menu-rounded.png" class="hamburger_menu"  alt="menu button">
+            </div>
+                <div class="top-nav">
+                    <ul>
+                    <li><a href="#" target="_blank"><i class="fa-brands fa-facebook-f"></i></a></li>
+                    <li><a href="#" target="_blank"><i class="fa-brands fa-twitter"></i></a></li>
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">LogOut</a></li>
+                    </ul>
+                </div>
+            
+                <nav class="navigation" id="nav">
+                    <div class="btn_wrapper">
+                        <button class="close_btn" onclick="closeMobileMenu()">
+                            <img src="images/closeBtn.png" alt="menu_btn">
+                        </button>
+                    </div>
+                    <div class="nav-logo"onclick="closeMobileMenu()" >
+                        <a href="./index.html"><img src="images/logo-removebg-preview.png" class="nav-logo" alt="logo of Orbit Web Innovations"></a>
+                    </div>
+                    <ul class="nav_wrapper">
+                        <li class="nav_items" onclick="closeMobileMenu()" >
+                            <a href="./about.html" class="linkA">About</a> 
+                        </li>
+                        <li class="nav_items" onclick="closeMobileMenu()">
+                            <a href="#courses" class="link">Program</a>
+                        </li>
+                        <li class="nav_items" onclick="closeMobileMenu()" >
+                            <a href="#courses" class="link">Courses</a>
+                        </li>
+                        <li class="nav_items" onclick="closeMobileMenu()">
+                            <a href="#lecturers" class="link">Lecturers</a>
+                        </li>
+                        <li class="nav_items" onclick="closeMobileMenu()" >
+                            <a href="#partners" class="link">Partners</a>
+                        </li>
+                        <li class="nav_items" onclick="closeMobileMenu()">
+                            <a href="#" class="linkB">OW Inno.</a>
+                        </li>
+                    </ul>
+                </nav>
+                <div class="nav_wrapper">
+                    <nav class="desktop_nav">
```

Comment:
- [ ] Amazing job with the layout and responsiveness. Let's just tweak some minor details to make the website a visual copy of the design. 🤝️ Please make the following changes on the about page too.
Kindly adjust the width of the navigation bar to match the design template.
I also suggest you add some padding to the button with borders. 😎️

**Your Design**
![Screenshot from 2023-02-08 10-52-57](https://user-images.githubusercontent.com/17243325/217482506-1b0eec09-6354-4569-acfb-66e358cf57e7.png)

**Template**
![Screenshot from 2023-02-08 10-52-21](https://user-images.githubusercontent.com/17243325/217482619-5fbb90f2-58f6-4965-9473-cf8c641f8186.png)

**Button**
| Your design                                                             | Template                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/217483308-c023ad58-7f68-4c38-845e-4231eed7e6bf.png" width="100">|<img src="https://user-images.githubusercontent.com/17243325/217483296-9f957c1b-8a41-4259-a144-7e1548bf6abf.png" width="100">|